desc/objectives/linear_objectives.py

```diff
@@ -831,9 +832,7 @@ class FixAxisR(_Objective):
 
     """
 
-    _scalar = False
-    _linear = True
-    _fixed = False
```

Comment:
`_fixed=False` but the parent objective has `_fixed=True`, does this affect the factorize_linear_constraints at all?

Reply:
It should have been fixed=True after the changes in #458 but wasn't updated. The `fixed` flag isn't actually used anywhere in the code currently, but I figure keep it for now until we decide on a more permanent fix.